File: src/codes/Users_Manual/segment/segment_change_transition.cpp
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~auto autd =
//~autd3::ControllerBuilder({autd3::AUTD3(autd3::Vector3::Zero())}).open(autd3::link::Nop::builder());
autd.send(autd3::SwapSegment::Modulation(autd3::Segment::S1,
                                         autd3::TransitionMode::Immediate));
//~return 0; }

File: src/codes/Users_Manual/segment/segment_change_transition.py
~from pyautd3 import Segment, SwapSegment
~from pyautd3 import Controller, AUTD3, TransitionMode
~from pyautd3.link.audit import Audit
~autd: Controller[Audit] = Controller.builder([AUTD3([0.0, 0.0, 0.0])]).open(Audit.builder())
autd.send(SwapSegment.Modulation(Segment.S1, TransitionMode.Immediate))